api/pkg/s3/bucketput.go

```diff
@@ -75,7 +78,12 @@ func (s *APIService) BucketPut(request *restful.Request, response *restful.Respo
 			return
 		}
 
-		backendName := createBucketConf.LocationConstraint
+		var backendName string
+		if isTier == "True" {
```

Comment:
Pls make the check case-insensitive, to handle "true" / True / TRUE etc..

Reply:
Done. Now it's case-insensitive. Thanks